package.json

```diff
@@ -50,7 +50,7 @@
   },
   "scripts": {
     "start": "cross-env DISABLE_ESLINT_PLUGIN=true craco start",
-    "build": "cross-env DISABLE_ESLINT_PLUGIN=true craco build",
+    "build": "npm run generate:template-parameters && cross-env DISABLE_ESLINT_PLUGIN=true craco build",
```

Comment:
Is this change for the docker build problem? How come it was moved from postinstall?

Reply:
Yeah, docker build process is broken (in master) because postinstall runs also in stage when we have copied only `package.json` and `package-lock.json` files -> https://github.com/asyncapi/studio/blob/master/Dockerfile#L9.

On `building` stage we have every needed deps/files to perform mentioned script :)

it was my mistake to add it in the `postinstall` hook.